test/src/041-rocache/main

```diff
@@ -1,37 +1,74 @@
 
 cvmfs_test_name="Read-only cache"
 
+lazy_umount() {
+  local dir=$1
+  if running_on_osx; then
+    sudo umount "$dir"
```

Comment:
It is used in the cleaup() function for both linux and OSX in this very file, but it can't be a lazy unmount, so instead it's a normal one